tests/parametric/test_span_links.py

```diff
@@ -203,8 +229,8 @@ def test_span_with_attached_links(self, test_agent, test_library):
         assert link.get("trace_id") == first.get("trace_id")
         assert (link.get("trace_id_high") or 0) == int(root_tid, 16)
         assert len(link.get("attributes")) == 4
-        assert link["attributes"].get("bools.0") == "true"
-        assert link["attributes"].get("bools.1") == "false"
+        assert link["attributes"].get("bools.0") == "True"
+        assert link["attributes"].get("bools.1") == "False"
```

Comment:
I don't think we should do that. We already had this discussion with Bob I think

Reply:
Ahhh interesting. So we are no longer flattening attributes in span links. We need to update the RFC: https://docs.google.com/document/d/1RcTjO8ER5FNweJ0rqVcQEbf6tb1PrlPsQ6coZZww8Xg/edit?disco=AAAA_TaDBkU. 

@bwoebi  The functionality was implemented in the python tracer last week. Should we revert this or should we support both approaches?

Reply:
I.e. we are taking a piece from the OTel textbook here, which is mandating a json-compatible representation, and in json it's "true" and not "True".

Reply:
I opened this fix for the python tracer: https://github.com/DataDog/dd-trace-py/pull/8123. Once this fix is merged the 4/6 system tests should pass